HackIllinois/ViewControllers/HILeaderboardViewController.swift

```diff
@@ -20,7 +20,7 @@ class HILeaderboardViewController: HILeaderboardListViewController {
         let fetchRequest: NSFetchRequest<LeaderboardProfile> = LeaderboardProfile.fetchRequest()
 
         fetchRequest.sortDescriptors = [
-            NSSortDescriptor(key: "id", ascending: true)
+            NSSortDescriptor(key: "points", ascending: false)
```

Comment:
Can you add a sort descriptor for the firstName? When multiple people have the same number of points, the leaderboard seems to display people in a different order during every refresh.

`NSSortDescriptor(key: "firstName", ascending: true)
`